src/py42/_internal/clients/detection_list_user.py

```diff
@@ -14,13 +15,33 @@ class DetectionListUserClient(BaseClient):
     _uri_prefix = u"/svc/api/{0}".format(_api_version)
     _resource = u"/user"
 
-    def __init__(self, session, user_context):
+    def __init__(self, session, user_context, user_client):
         super(DetectionListUserClient, self).__init__(session)
         self._user_context = user_context
+        self._user_client = user_client
 
     def _make_uri(self, action):
         return u"{0}{1}{2}".format(self._uri_prefix, self._resource, action)
 
+    def create_if_not_exists(self, user_id):
+        """Find whether detection list profile exists for given uid. If not, if user_id is a
```

Comment:
Have chance to revise this to something like: `Find _out_ whether _the_ detection list profile exists for _a_ given UID`.. Just missing a few articles, but no big deal!

Reply:
Changed.